runner.py

```diff
@@ -759,11 +759,11 @@ def setup_services(self):
                         raise RuntimeError(f"Docker container setup environment var key had wrong format. Only ^[A-Z_]+[A-Z0-9_]*$ allowed: {env_key} - Maybe consider using --allow-unsafe or --skip-unsafe")
 
                     if not self._allow_unsafe and \
-                        re.search(r'^[a-zA-Z0-9_]+[a-zA-Z0-9_-]*$', env_value) is None:
+                        re.search(r'^[a-zA-Z0-9_]{1}[a-zA-Z0-9_.:/-]{0,1023}$', env_value) is None:
```

Comment:
Maybe we should change this to a block list instead of allow list, so we don't need to revisit this in the near future?

Reply:
I have created an alternative PR that changes it to a blacklist approach: #577